src/main/java/com/minecolonies/coremod/util/WorkerUtil.java

```diff
@@ -61,6 +62,12 @@
      * List of tools to test blocks against, used for finding right tool.
      */
     public static List<Tuple<ToolType, ItemStack>> tools;
+    
+    /**
+     * A list of all the path blocks on which the citizen walks faster.
+     */
+    private static final List<Block> pathBlocks = Arrays.asList(new Block[] {Blocks.GRAVEL, Blocks.GRASS_PATH,
+          Blocks.STONE_BRICKS, Blocks.STONE_BRICK_STAIRS, Blocks.STONE_BRICK_SLAB});
 
```

Comment:
final List<Block> pathBlocks = Arrays.asList(Blocks.GRAVEL, Blocks.GRASS_PATH,
          Blocks.STONE_BRICKS, Blocks.STONE_BRICK_STAIRS, Blocks.STONE_BRICK_SLAB);

Is enough, no need for new Block[] {}

Reply:
Thanks, didin't realize.